src/main.ts

```diff
@@ -0,0 +1,49 @@
+import { Command, Helper, OptionsHelper } from '@dojo/interfaces/cli';
+import * as webpack from 'webpack';
+
+import devConfigFactory from './dev.config';
+import testConfigFactory from './test.config';
+import distConfigFactory from './dist.config';
+import logger from './logger';
+
+const fixMultipleWatchTrigger = require('webpack-mild-compile');
+
+const command: Command = {
+	group: 'build',
+	name: 'app',
+	description: 'create a build of your application',
+	register(options: OptionsHelper) {
+		options('mode', {
+			describe: 'the output mode',
+			alias: 'm',
+			default: 'dist',
+			choices: ['dist', 'dev', 'test']
+		});
+	},
+	run(helper: Helper, args: any) {
+		console.log = () => {};
```

Comment:
Is this needed for the `logger` to function properly?

Reply:
It ensures that the output is always consistent and there's no random output from the build pipeline.